sonar-iac-plugin/src/main/resources/static/documentation.md

```diff
@@ -1,22 +1,23 @@
 ---
-title: Infrastructure-as-Code
+title: Terraform/CloudFormation
 key: iac
 ---
 
 <!-- static -->
 <!-- update_center:iac -->
 <!-- /static -->
 
-
-
 ## Language-Specific Properties
 
-Discover and update the language-specific [properties](/analysis/analysis-parameters/)
+Discover and update the CloudFormation [properties](/analysis/analysis-parameters/) **<!-- sonarcloud -->Project <!-- /sonarcloud -->[Administration > General Settings > Cloudformation](/#sonarqube-admin#/admin/settings?category=cloudformation)**
 
-Cloudformation-specific properties in: <!-- sonarcloud -->Project <!-- /sonarcloud -->**[Administration > General Settings > Cloudformation](/#sonarqube-admin#/admin/settings?category=cloudformation)**
+Discover and update the Terraform [properties](/analysis/analysis-parameters/) **<!-- sonarcloud -->Project <!-- /sonarcloud -->[Administration > General Settings > Terraform](/#sonarqube-admin#/admin/settings?category=terraform)**
 
-Terraform-specific properties in: <!-- sonarcloud -->Project <!-- /sonarcloud -->**[Administration > General Settings > Terraform](/#sonarqube-admin#/admin/settings?category=terraform)**
+## Supported Versions, Formats and Providers
+* Terraform 1 (HCL/Terraform only)
+* Cloudformation with AWSTemplateFormatVersion 2010-09-09 (YAML and JSON)
+* AWS, Azure and GCP
 
 ## External Analyzers
```

Comment:
This section should be named: "Related Pages"

Reply:
Also fine for me.